app/jobs/decidim/direct_verifications/authorize_users_job.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "decidim/direct_verifications/instrumenter"
+
+module Decidim
+  module DirectVerifications
+    class AuthorizeUsersJob < BaseImportJob
+      class NullSession; end
+
+      def process_users
+        emails.each do |email, data|
+          AuthorizeUser.new(email, data, session, organization, instrumenter).call
+        end
+      end
+
+      def send_email_notification
```

Comment:
We could move this to the command

Reply:
Done in https://github.com/coopdevs/decidim-verifications-direct_verifications/pull/15